zzk/service/hoststate.go

```diff
@@ -157,17 +159,14 @@ func (l *HostStateListener) Spawn(shutdown <-chan interface{}, stateID string) {
 
 		// load the service state node
 		sspth := path.Join("/services", serviceID, stateID)
-		exists, err := l.conn.Exists(sspth)
-		if !exists {
-			logger.Debug("Service state was removed, exiting")
+		ssdat := &ServiceState{}
```

Comment:
This needs to be loaded outside of the for loop, since this method is the only place where this node is ever modified